gnosis/abstract-semantics.k

```diff
@@ -36,7 +36,7 @@ module ABSTRACT-SEMANTICS
 
    rule <k> Ccallgas(SCHED, ISEMPTY:Bool, GCAP, GAVAIL, VALUE)
           => #gas(Cgascap(SCHED, GCAP, GAVAIL, Cextra(SCHED, VALUE, ISEMPTY)) +Int Gcallstipend < SCHED >, 0, 0) ... </k>
-     requires VALUE =/=Int 0
+     requires 0 <Int VALUE
```

Comment:
this is better to have since it ensures that we have a sufficient pre-condition.